src/utils.ts

```diff
@@ -0,0 +1,138 @@
+import { PartialFileList } from './PartialFile';
+
+export type StringObject = Record<string, string>;
+
+/**
+ * Type for [[`groupFiles`]] function
+ */
+export type GroupFiles = (
```

Comment:
this type does not bring a lot (and add some confusion for me between Groupfiles and GroupesFiled), just remove it and declare the type of the parameters directly in the function.